llvm/lib/Target/AMDGPU/GCNSubtarget.h

```diff
@@ -600,6 +601,10 @@ class GCNSubtarget final : public AMDGPUGenSubtargetInfo,
     return UnalignedAccessMode;
   }
 
+  bool requiresNaturallyAlignedBufferAccess() const {
```

Comment:
nit: might be better to name it as `hasXXXXX`.

Reply:
Right, I wasn't sure how to phrase it, and there is already a precedent for "require" (RequiresCOV). HasNaturalAlignmentRequirementBufferAccess?

Reply:
I guess you can directly use `hasRequireNaturallyAlignedBufferAccess`. Grammar is not that important here. :-)

Reply:
Done.